Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/ISISIndirectDiffractionReduction.py

```diff
@@ -341,16 +333,7 @@ def _setup(self):
         logger.information('IPF filename is: %s' % self._ipf_filename)
 
         # Only enable sum files if we actually have more than one file
-        sum_files = self.getProperty('SumFiles').value
-        self._sum_files = False
-
-        if sum_files:
-            num_raw_files = len(self._data_files)
-            if num_raw_files > 1:
-                self._sum_files = True
-                logger.information('Summing files enabled (have %d files)' % num_raw_files)
-            else:
-                logger.information('SumFiles options is ignored when only one file is provided')
```

Comment:
I'm not entirely sure about this warning, SumFiles isn't really 'ignored' it just has no additional effect due to summing across one run is the same as the run itself.

Reply:
True, it would be better worded as "has no effect". I'm not sure how useful it is but maybe it would help in a few edge case scenarios, e.g. if you enter `12345-12345` when you meant to enter a proper range.